calico-cloud/image-assurance/configuring-the-runtime-view.mdx

```diff
@@ -0,0 +1,27 @@
+---
+description: Use the Runtime View to find vulnerabilities in your cluster.
+---
+
+# Configuring the Runtime View
+
+:::warning
+
+The Runtime View requires that all the connected Managed Clusters are running Kubernetes using a CRI for container management.
```

Comment:
Maybe state kubernetes versions and link to something explaining what the CRI is.